### PR TITLE
Random Events can no longer trigger on alert levels above red

### PR DESCRIPTION
**Pull request by Bibby0110**
# Github documenting your Pull Request

#12071 alternative

# Wiki Documentation

# Changelog

:cl:  
tweak: Random events no longer trigger on 
/:cl:

### CODE REVIEW

**Comment by Xoxeyos** [comment]
c3JROGr
I don't believe this should be implemented, we already have a tool that allows us to toggle events, ghost roles, and other such circumstances from spawning, using the 'Ghost Pool Protection'.

**Comment by Bibby0110** [comment]
> c3JROGr
> I don't believe this should be implemented, we already have a tool that allows us to toggle events, ghost roles, and other such circumstances from spawning, using the 'Ghost Pool Protection'.

That's an active choice.

This makes it passively happen.

**Comment by Xoxeyos** [comment]
> 
> 
> > c3JROGr
> > I don't believe this should be implemented, we already have a tool that allows us to toggle events, ghost roles, and other such circumstances from spawning, using the 'Ghost Pool Protection'.
> 
> That's an active choice.
> 
> This makes it passively happen.

Why should it be set to disallow events above red at all times? How is this objectively better?

**Comment by adamsong** [comment]
Ideally the game can run without admin intervention, because there might not be admins online

**Comment by Xoxeyos** [comment]
> 
> 
> Ideally the game can run without admin intervention, because there might not be admins online

No alerts above delta can be triggered without an admin online, and the only case where a delta alert happens is when a nuke is activated, in which case everything is already fucked up.

**Comment by adamsong** [comment]
Malf AI can trigger delta, and that is a pretty long timer

**Comment by Xoxeyos** [comment]
> 
> 
> Malf AI can trigger delta, and that is a pretty long timer

You're right, however I'd still like to be able to turn the random events on and off at will, and I, and perhaps others, would not like to dig around in VV trying to turn it off, when they want it changed.

**Comment by adamsong** [comment]
This doesn't remove that

**Review by JamieD1 — APPROVED** [review]

File: code/modules/events/_event.dm [hunk 1]
```diff
@@ -62,6 +64,8 @@
 		return FALSE
 	if(ispath(typepath, /datum/round_event/ghost_role) && !(GLOB.ghost_role_flags & GHOSTROLE_MIDROUND_EVENT))
 		return FALSE
+	if(GLOB.security_level >= SEC_LEVEL_GAMMA && !security_level_override)
+    return FALSE
```
Comment (JamieD1) [inline code/modules/events/_event.dm]:
```suggestion
	if(GLOB.security_level >= SEC_LEVEL_GAMMA && !security_level_override)
		return FALSE
```

**Comment by adamsong** [comment]
#12071 was merged

**Pull request closed without merging**